lib/rest/RequestHandler.js

```diff
@@ -86,6 +86,9 @@ class RequestHandler {
                     if(auth) {
                         headers.Authorization = this.#client._token;
                     }
+                    if(this.options.headers) {
+                        Object.assign(headers, this.options.headers);
+                    }
```

Comment:
Maybe it would be a bit better putting in all the custom headers before authorization info and others are put in?

```suggestion
                    if(this.options.headers) {
                        Object.assign(headers, this.options.headers);
                    }
                    if(auth) {
                        headers.Authorization = this.#client._token;
                    }
```

Reply:
maybe, it can prevent accidentally updating `Authorization` property. Also, a warning can be added to description in `Client.js`. Such as `Authorization key is used for bot token.` etc.